apps/daffio/src/app/content/home/home-routing.module.ts

```diff
@@ -5,15 +5,21 @@ import {
 }  from '@angular/router';
 
 import { DaffioHomeViewComponent } from './view/home-view.component';
+import { DaffioMarketingHeaderContainer } from '../../core/header/containers/marketing-header/marketing-header.component';
+import { DaffioRouterNamedViewsEnum } from '../../core/router/named-view/models/named-views.enum';
+import { RouteWithNamedViews } from '../../core/router/named-view/models/route.type';
 
 const homepage: Routes = [
-  {
+  <RouteWithNamedViews>{
     path: '',
     pathMatch: 'full',
     component: DaffioHomeViewComponent,
     data: {
       title: 'Modern front-end development toolkit for ecommerce PWAs',
       description: 'Build your store with ease and flexibility — take advantage of the cutting edge with Angular (6+), Redux and Progressive Web Apps.',
+      namedViews: {

```

Comment:
This implementation means that we'll need to repeat this in all the content pages, like `/why-pwa` and `/support`? Is this the only way or can we batch apply the same nav to a set of pages?

Reply:
The entire tree of the current route config will be searched for named views so you can specify a "default" named view in the [template routing config](https://github.com/graycoreio/daffodil/blob/ab2836b1b2c72189b74285d38455b9f237b0b9dd/apps/daffio/src/app/app-routing.module.ts#L11) if you want (or any other parent route). More deeply nested named views will take precedence over higher ones. I actually need to add a test case for this; I didn't test this specific use case. LMK how this works for you.

Reply:
So if we specific `DaffioMarketingHeaderContainer` as the default, how can we set a separate default to all `/docs` related pages? Or would it make more sense to set the docs header as the default since there are less marketing pages?

Reply:
Its all a matter of the route config nesting hierarchy. If you want a shared named view, set it in a common parent route.

I added an example of how to set a default named view and also how to disable it for a child route. Let me know if you have more questions.

Reply:
I changed the implementation a bit to achieve what it probably closer to the final product. It showcases how to group routes for the purposes of setting a common named view